tensorflow/compiler/xla/service/hlo_instruction.cc

```diff
@@ -2158,9 +2158,15 @@ string HloInstruction::ToStringWithCanonicalNameMap(
     StrAppend(&result, PrintName(name(), options), " = ");
   }
 
-  // Print opcode, operand(s) and shape.
-  StrAppend(&result, ShapeUtil::HumanStringWithLayout(shape()), " ",
-            HloOpcodeString(opcode()), "(",
+  // Print shape
```

Comment:
thanks for the review.  updated with the full stop.

i had not considered any other scheme.  we're not interested in layout because it doesn't apply on our hardware, so it makes sense to reduce the visual clutter by eliminating it alltogether.